### PR TITLE
SDKS-1965 Update Flutter Plugin with Push enhancements

### PR DESCRIPTION
**Pull request by rodrigoareis**
# JIRA Ticket

[SDKS-1965](https://bugster.forgerock.org/jira/browse/SDKS-1965)

# Description

The Push enhancements available on AM 7.2 and latest versions of the Android and iOS SDK are now added to the Flutter Plugin.

**NOTE: This PR should not be merged until the release of the Android SDK 3.3.4**

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

File: forgerock-authenticator/CHANGELOG.md [hunk 1]
```diff
@@ -4,3 +4,14 @@
 
 #### Added
 - Initial release of ForgeRock Authenticator for Flutter
+
+## [1.1.0]
```
Comment (spetrov) [inline forgerock-authenticator/CHANGELOG.md:8]: The new features and releases should appear on top of the page... at least all our other repos follow this convention.

**Review by spetrov — COMMENTED** [review]
Looks good to me! Have a few minor comments only.

File: forgerock-authenticator/android/build.gradle [hunk 2]
```diff
@@ -40,7 +40,15 @@ dependencies {
...
-    implementation 'org.forgerock:forgerock-authenticator:3.3.3'
-    implementation 'org.forgerock:forgerock-core:3.3.3'
+//    implementation 'org.forgerock:forgerock-authenticator:3.3.3'
+//    implementation 'org.forgerock:forgerock-core:3.3.3'
+
+    //add lib via aar-depency
+    compile(name: 'forgerock-authenticator-debug', ext: 'aar')
+    compile(name: 'forgerock-core-debug', ext: 'aar')
+    implementation 'com.nimbusds:nimbus-jose-jwt:9.0.1'
+    implementation 'com.squareup.okhttp3:okhttp:4.3.1'
```
[the remark this answers is not in the document]
Comment (rodrigoareis) [reply forgerock-authenticator/android/build.gradle:50]: Yes, we should update it, but those are temporary and will be removed on we release Android SDK 3.3.4

File: forgerock-authenticator/android/build.gradle [hunk 3]
```diff
@@ -40,7 +40,15 @@ dependencies {
...
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.3.1'
```
[the remark this answers is not in the document]
Comment (rodrigoareis) [reply forgerock-authenticator/android/build.gradle:51]: Same as okttp3

File: forgerock-authenticator/CHANGELOG.md (hunk 1 above)
Comment (rodrigoareis) [reply forgerock-authenticator/CHANGELOG.md:3]: Nice catch! Done

**Review by spetrov — APPROVED** [review]

**Pull request merged**